utils/lvm2/Makefile

```diff
@@ -9,14 +9,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=LVM2
-PKG_VERSION:=2.02.177
-PKG_RELEASE:=2
+PKG_VERSION:=2.02.180
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0 LGPL-2.1
 
 PKG_SOURCE:=$(PKG_NAME).$(PKG_VERSION).tgz
-PKG_SOURCE_URL:=ftp://sources.redhat.com/pub/lvm2/releases \
-		http://ftp.gwdg.de/pub/linux/sources.redhat.com/lvm2/
-PKG_HASH:=4025a23ec9b15c2cb7486d151c29dc953b75efc4d452cfe9dbbc7c0fac8e80f2
+PKG_SOURCE_URL:=https://ftp.gwdg.de/pub/linux/sources.redhat.com/lvm2 \
+		ftp://sources.redhat.com/pub/lvm2/releases
```

Comment:
Please put `sources.redhat.com` first as they have several mirrors.

Reply:
Ah looks like it's reachable over HTTP but not HTTPS. I'll remove the ftp.